topology/probes/nsm/nsm.go

```diff
@@ -32,7 +32,7 @@ import (
 	cc "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/crossconnect"
 	localconn "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/local/connection"
 	remoteconn "github.com/networkservicemesh/networkservicemesh/controlplane/pkg/apis/remote/connection"
-	v1 "github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis/networkservice/v1"
+	v1 "github.com/networkservicemesh/networkservicemesh/k8s/pkg/apis/networkservice/v1alpha1"
 	"github.com/networkservicemesh/networkservicemesh/k8s/pkg/networkservice/clientset/versioned"
```

Comment:
i don't think mapping v1alpha1 to v1 is a good practice, rather you should leave default mapping to v1aloha1 as you may at some point have to support both v1 and v1alpha1 ...